config/jobs/image-pushing/k8s-staging-kops.yaml

```diff
@@ -19,7 +19,8 @@ postsubmits:
               - --project=k8s-staging-kops
               - --scratch-bucket=gs://k8s-staging-kops-gcb
               - --env-passthrough=PULL_BASE_REF
-              - ci/postsubmit/push-to-staging/
+              - --gcb-config=ci/postsubmit/push-to-staging/cloudbuild.yaml
+              - .
```

Comment:
I think the preferred way to accomplish the desired effect is this:

```yaml
- --build-dir=.
- ci/postsubmit/push-to-staging/
```

The intent with `--gcb-config` is for renames in the same directory (though this will probably work as is, it may violate an assumption in the future)